From a55d55f6f59c0f366d21d649c3486504bb03fae0 Mon Sep 17 00:00:00 2001
From: Andrew Wang <wang@apache.org>
Date: Fri, 19 Feb 2016 16:33:17 -0800
Subject: [PATCH 1252/2848] MAPREDUCE-6637. Testcase Failure :
 TestFileInputFormat.testSplitLocationInfo.
 Contributed by Brahma Reddy Battula.

(cherry picked from commit 37577852ba54ba506b0b6c4db5e95d176f5e9b63)
(cherry picked from commit 7f6737951a124480af2f3fff9358865fff3ed091)
(cherry picked from commit 403934c0115c12e895509ce5321046d3af731377)

Change-Id: I1920783e914024f36ef08661bfc64f297d23e2ce
---
 .../apache/hadoop/mapred/TestFileInputFormat.java  |    2 +-
 .../mapreduce/lib/input/TestFileInputFormat.java   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapred/TestFileInputFormat.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapred/TestFileInputFormat.java
index ba636b6..141086d 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapred/TestFileInputFormat.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapred/TestFileInputFormat.java
@@ -243,7 +243,7 @@ private Configuration getConfiguration() {
     }
 
     @Override
-    public BlockLocation[] getFileBlockLocations(Path p, long start, long len)
+    public BlockLocation[] getFileBlockLocations(FileStatus file, long start, long len)
         throws IOException {
       return new BlockLocation[] {
           new BlockLocation(new String[] { "localhost:50010", "otherhost:50010" },
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/input/TestFileInputFormat.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/input/TestFileInputFormat.java
index 3f877f1..dff0e0d 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/input/TestFileInputFormat.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/input/TestFileInputFormat.java
@@ -422,7 +422,7 @@ private Configuration getConfiguration() {
     }
 
     @Override
-    public BlockLocation[] getFileBlockLocations(Path p, long start, long len)
+    public BlockLocation[] getFileBlockLocations(FileStatus file, long start, long len)
         throws IOException {
       return new BlockLocation[] {
           new BlockLocation(new String[] { "localhost:50010", "otherhost:50010" },
-- 
1.7.9.5

